cdc_replication_test.py

```diff
@@ -197,6 +212,23 @@ def test_replication(self) -> None:
             self.log.info('Consistency check successful.')
         else:
             self.collect_data_for_analysis(
-                    migrate_log_path, replicator_log_path,
-                    master_node, replica_node)
+                migrate_log_path, replicator_log_path,
+                master_node, replica_node)
             self.fail('Consistency check failed.')
+
+    def get_email_data(self):
```

Comment:
Where is this function used?

Reply:
@kbr- it is required to build email report